### PR TITLE
LPS-103653 Remove portal-kernel-*-compat | rebase and resend 2

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
SF failures are irrelevant.

ci:test:relevant

**Comment by dantewang** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/dependencies/groovy-script-portal-cache-put-modules.groovy [hunk 1]
```diff
@@ -18,4 +18,4 @@ try {
 }
 catch (Exception e) {
 	out.println(e);
-}
\ No newline at end of file
+}
```
Comment (tinatian) [inline portal-web/test/functional/com/liferay/portalweb/dependencies/groovy-script-portal-cache-put-modules.groovy:21]: @dantewang 
Why this is needed? It seems just adding an empty line, is this requested by SF?

Tina

**Review by tinatian — COMMENTED** [review]
Hi @dantewang ,
Why this commit message "LPS-103653 (LPS-89288) Also remove other portal-kernel-*-compat modules" contain two tickets number? 

Tina.

**Comment by dantewang** [comment]
@tinatian LPS-89288 was a previous ticket created by Preston on removing these modules.

**Pull request closed without merging**